extensions/amp-story/1.0/_locales/en.json

```diff
@@ -246,5 +246,9 @@
   "88": {
     "string": "Skip next",
     "description": "Label for a button that advances to the next element in the carousel"
+  },
+  "89": {
+    "string": "Your response will be sent to ",
+    "description": "Text that explains where the response will be sent on quizzes and polls disclaimers, ending with space"
```

Comment:
This lacks context for a translator.
They don't know where/how the text will be displayed: should they be very concise? Is it okay to use more words to keep more context?
They don't know what "quizzes and polls disclaimers" are.
This should not end with a space. This is confusing at best, and we don't know if the software they use, or linters, even allow for that.

Another example: "Text displayed to users after clicking a button that reveals a disclaimer, telling them more about where their user data will be stored, after they interact with a poll or quiz."

Reply:
I agree the text you suggest is more clear.

I'd say there's something very important about including the space at the end of the translated string in English, which is that "to" as a proposition is a separate word in English but not in Hebrew for instance, where "to" is a prefix of the object (eg: "to him" is just one word, imagine "tohim").

If technically it's a problem, we could add a margin or add it in some other way, but having a space before the entity is not a generic guarantee in all languages. I wonder if it would make sense to introduce a substitution/format to add the Entity in case in other languages it wouldn't conform to being last (eg: `"Your response will be sent to %s"`) That's why having the space as part of the translation seemed better than not. WDYT?

Reply:
Well, space seems to be provided for free on inline elements so fighting against that is not a good idea, will follow your suggestion